backend/opre_ops/django_config/settings/common.py

```diff
@@ -87,3 +87,11 @@
 # https://docs.djangoproject.com/en/3.2/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+REST_FRAMEWORK = {
+    # Use Django's standard `django.contrib.auth` permissions,
+    # or allow read-only access for unauthenticated users.
+    "DEFAULT_PERMISSION_CLASSES": [
+        "rest_framework.permissions.DjangoModelPermissionsOrAnonReadOnly"
+    ]
+}
```

Comment:
What's this for?

Reply:
For the built-in DRF whether to allow Anonymous users to actually call endpoints to make changes. Was a basic "default" permission I thought it could use out of the box. It'll chance as we implement our own users/permissions.

Reply:
Oh, cool.  Sounds good!